Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Simulation/ScaleFadeAndDestroy.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    public class ScaleFadeAndDestroy : MonoBehaviour
```

Comment:
Can you explain what this class does?

Reply:
This is an effect script that that will fade and scale a gameObject over the Lifespan then destroy itself at the end. I use it to provide interaction feedback. Like OnClick and burst will happen.

Reply:
removed